index.bs

```diff
@@ -398,7 +398,7 @@ On getting, the {{NavigatorUAData/mobile}} attribute must return the [=user agen
 
 The <dfn method for="NavigatorUA"><code>getHighEntropyValues(|hints|)</code></dfn> method MUST run these steps:
 
-1. Let |p| be a [=a new promise=].
+1. Let |p| be a [=a new promise=] created with [=current settings object=]'s [=environment settings object/realm=].
```

Comment:
```suggestion
1. Let |p| be a [=a new promise=] created in [=the current Realm=].
```

This may necessitate adding an anchors block entry for https://tc39.es/ecma262/#current-realm since ES spec/Bikeshed integration is still sad.

Reply:
Added a direct link, so that this won't be blocking. Happy to also file a PR for adding that to ES, but would appreciate some pointers

Reply:
Nah, the ES spec won't accept PRs to add Bikeshed-isms, sadly. You have to work around it :(.

A direct link is reasonable, although if you're going to link somewhere more than once you could use the anchors block instead: cf. https://github.com/whatwg/streams/blob/webidl-rewrite/index.bs#L37

Reply:
Makes sense, thanks! :)